### ISSUE TITLE
[Question]: How do I disable IDFA

### ISSUE THREAD

**Issue by cainaleaouk** [SudoPlz/react-native-amplitude-analytics#57]
Hi there,

as the title says, I'd like to disable IDFA collection since our app cannot use it. Is there a way to do something similar to this?
https://developers.amplitude.com/docs/disabling-automatic-tracking-of-properties#disabling-tracking-on-ios

Cheers.

**Comment by SudoPlz** [comment]
Not currently, but you're super welcome to open a PR that adds that functionality.

**Comment by ilyakar** [comment]
Same. We have a kid's app and all references to IDFA-related stuff have to be gone.

**Comment by Alex123456780** [comment]
Amplitude uses `IDFV` by default and not `IDFA`. See FAQ [here](https://help.amplitude.com/hc/en-us/articles/360047228311-FAQ-Apple-s-iOS-14-IDFA-Changes#does-amplitude-accept-idfa-identifier-for-advertisers-as-the-dev).

**Comment by ilyakar** [comment]
Not true. Apple rejections happen due to IDFA reasons. I turned to the JS SDK instead to avoid this problem.

**Comment by Alex123456780** [comment]
@ilyakar you can try to ask Amplitude's technical support about your problem (e.g. "i don't activate IDFA usage, your SDK [disables IDFA by default](https://help.amplitude.com/hc/en-us/articles/360047228311#does-amplitude-accept-idfa-identifier-for-advertisers-as-the-dev), but Apple rejects my app anyway").

**Comment by ilyakar** [comment]
I just switched to the Amplitude JS sdk and all is fine for me :)

It seems this SDK is referencing to the AdvertisingIdentifier, if not including the AdSupport.framework, and that’s causing problems for people with kids apps.

Kind regards,
Ilya
 <https://bowtie.mailbutler.io/tracking/hit/82780f1c-2fac-497b-abb1-6a29598e9bda/e24f1f68-3277-444e-a5ba-110cb8ba7558>
Product Manager
www.ilya.link <https://bowtie.mailbutler.io/tracking/hit/82780f1c-2fac-497b-abb1-6a29598e9bda/ef008d39-8597-4703-a6fe-b59daea34cc2> | LinkedIn <https://bowtie.mailbutler.io/tracking/hit/82780f1c-2fac-497b-abb1-6a29598e9bda/e24f1f68-3277-444e-a5ba-110cb8ba7558> | Twitter <https://bowtie.mailbutler.io/tracking/hit/82780f1c-2fac-497b-abb1-6a29598e9bda/31a3d654-7068-409c-b816-e09cf972ebcb>
